### DISCUSSION TITLE
PWA  and push notification

### DISCUSSION THREAD

**Discussion by ghost** [livewire/livewire#1235] [4. General]
Hello,
Can we have PWA and push notification with livewire?

**Comment by lancepioch** [comment]
Here is a good solution from Pusher: https://laravel-notification-channels.com/pusher-push-notifications/

**Comment by mblarsen** [comment]
Yes, for instance if you use https://beyondco.de/docs/laravel-websockets/getting-started/introduction and set up Laravel Echo to use pusher. It is super easy to get working.

Re: Livewire. Yes you can. [See here](https://laravel-livewire.com/docs/laravel-echo) for the syntax in your Livewire component to listen to events.
